plugins/event-watcher/pkg/handlers/audit_log_handler.go

```diff
@@ -26,6 +26,8 @@ type AuditLogHandler struct {
 	stopCh         chan struct{}
 }
 
+var alreadyProcessedAuditIDs = map[string]bool{}
```

Comment:
this is volatile data that will be lost in a restart or in case of multiple pods could be processed already by the other pod.

Ideally, the API should be able idempotent (ensures that making multiple identical requests produces the same result as a single request, without creating duplicates)

Reply:
I know, I want just to avoid a lot of unnecessary API calls